pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.poetry]
+name = "dcicwrangling"
+version = "0.1.0"
+description = "Scripts and Jupyter notebooks for 4DN wrangling"
+authors = ["4DN-DCIC Team <support@4dnucleome.org>"]
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/4dn-dcic/dcicwrangling"
+repository = "https://github.com/4dn-dcic/dcicwrangling"
+
+packages = [
+  { include="src", from="." }
+]
+
+[tool.poetry.dependencies]
+python = ">=3.6, <3.8"
+dcicutils = ">=0.31.1"
```

Comment:
I would recommend a caret requirement here rather than `>=`, or if you're going to use that at least add `<1` as well.